### DISCUSSION TITLE
Problem with the default tw-backdrop-brightness property

### DISCUSSION THREAD

**Discussion by sellmon** [tailwindlabs/tailwindcss#11600] [Help]
**What version of Tailwind CSS are you using?**

Version: "tailwindcss": "^3.3.2",

**What build tool (or framework if it abstracts the build tool) are you using?**

"next": "^13.4.7",
"turbo": "^1.10.6"

**What version of Node.js are you using?**

v18.14.2

**What browser are you using?**

Chrome 114.0.5735.198
In Firefox and other browser everything is fine.

**What operating system are you using?**

macOS

**Reproduction URL**

Can't provide.

**Describe your issue**

Hello everyone, I'm facing an issue, as you can see in the GIF.
![Jul-12-2023 22-06-25](https://github.com/tailwindlabs/tailwindcss/assets/108669343/9aa796fc-16bb-4eaa-8b00-893ee89b9fb8)

My stack includes NextJS, Tailwind, Turborepo, and pnpm.
In Chrome, I noticed that the tw-backdrop-brightness property is being generated, and it's causing display issues.

**Comment by thecrypticace** [comment]
Hey! This feels like a Chrome bug to me. ~~Are you seeing this in other browsers like Firefox?~~

Sorry, re-read. Yeah this is definitely a Chrome bug. I'm not sure how much help we can be without a reproduction though. Is there anything you can provide?

**Comment by sellmon** [comment]
Most likely not, as it could be a specific issue with the stack, as I already mentioned I have TurboRepo, NextJS, TS. I can create more GIF's 😅
Is there any way to manually disable this property?

**Comment by adamwathan** [comment]
You can definitely provide a reproduction, just create a new project just like yours, set up all the necessary pieces to trigger the issue, then publish that project to GitHub so we can clone it down and troubleshoot 🙂 Without that it’s impossible to help I’m afraid, we need to see the whole project.

**Comment by sellmon** [comment]
> You can definitely provide a reproduction, just create a new project just like yours, set up all the necessary pieces to trigger the issue, then publish that project to GitHub so we can clone it down and troubleshoot 🙂 Without that it’s impossible to help I’m afraid, we need to see the whole project.

Sure, when I have the time, I will try to create a new project and replicate the bug there
